src/stylesheets/core/_variables.scss

```diff
@@ -418,29 +418,29 @@ $tokens-color-theme: (
 );
 
 $tokens-color-state: (
-  "error-lighter": color($theme-color-error-lighter),
-  "error-light": color($theme-color-error-light),
-  "error": color($theme-color-error),
-  "error-dark": color($theme-color-error-dark),
-  "error-darker": color($theme-color-error-darker),
-  "warning-lighter": color($theme-color-warning-lighter),
-  "warning-light": color($theme-color-warning-light),
-  "warning": color($theme-color-warning),
-  "warning-dark": color($theme-color-warning-dark),
-  "warning-darker": color($theme-color-warning-darker),
-  "success-lighter": color($theme-color-success-lighter),
-  "success-light": color($theme-color-success-light),
-  "success": color($theme-color-success),
-  "success-dark": color($theme-color-success-dark),
-  "success-darker": color($theme-color-success-darker),
-  "info-lighter": color($theme-color-info-lighter),
-  "info-light": color($theme-color-info-light),
-  "info": color($theme-color-info),
-  "info-dark": color($theme-color-info-dark),
-  "info-darker": color($theme-color-info-darker),
-  "disabled-light": color($theme-color-disabled-light),
-  "disabled": color($theme-color-disabled),
-  "disabled-dark": color($theme-color-disabled-dark)
+  "error-lighter": color($theme-color-error-lighter, set-theme, no-warn),
```

Comment:
@thisisdano Wasn't totally sure why these had `no-warn`, but I kept these consistent with what I saw being done in `$tokens-color-theme`

Reply:
These have a special internal flag on the `color()` function to prevent multiple warnings when using a custom value. As long as you used the same structure as the others, it should work as expected.